projects/goharbor/harbor/helm/sedfile.template

```diff
@@ -0,0 +1,15 @@
+s/^version: 1.8.1$/version: ${IMAGE_TAG}-helm/
```

Comment:
Not for this PR, but we should consider being able to parameterize the GIT_TAG here so its not something we need update when bumping versions. could be easily forgotten.